### PR TITLE
Mes 2136 - Display submission status of test

### PR DESCRIPTION
**Pull request by Cavalier07**
## Description and relevant Jira numbers
- Add upload pending flag test slots
- Add test submitted flag to test slots

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [ ] branch rebased against the latest develop
- [ ] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [ ] Reviewers selected in Github
- [x] Any new reusable component/widget added to component library on Confluence

- [x] Screenshot(s) captured on the physical device (if applicable)
- [x] Tested by QA
- [x] PO's approval

IMG_0003

### CODE REVIEW

**Comment by ammarhaiderbjss** [comment]
@Cavalier07 Is the activity code to be added later?

File: src/providers/test-submission/test-submission.ts [hunk 1]
```diff
@@ -13,9 +14,12 @@ export class TestSubmissionProvider {
     ) {}
 
   submitTests = (data: StandardCarTestCATBSchema[]): Observable<any> => {
+
+    const compressedData = gzipSync(JSON.stringify(data));
```
Comment (ammarhaiderbjss) [inline src/providers/test-submission/test-submission.ts]: Quick unit test to cover this?

Comment (Cavalier07) [reply src/providers/test-submission/test-submission.ts]: I'm pulling this out to a new PR as it's not working how i'd like.

**Comment by Cavalier07** [comment]
> @Cavalier07 Is the activity code to be added later?

I assume there is a ticket to do that. There's some code there to display the code it needs wiring up

**Pull request merged**